README.md

```diff
@@ -36,7 +36,7 @@ Demo available [here](http://florianbelhomme.com/flobfoundationdemobundle/).
 - [ ] Icon Bar
 - [ ] Support for off-canvas
 - [ ] Modal
-- [ ] Icon with labels
+- [X] Icon with labels
```

Comment:
Please remove this check, i will manage the list.
I was talking about thoses labels : http://foundation.zurb.com/docs/components/labels.html